src/common/utils/channel/message-channel-injection-token.ts

```diff
@@ -4,9 +4,9 @@
  */
 
 import { getInjectionToken } from "@ogre-tools/injectable";
-import type { JsonValue } from "type-fest";
+import type { IpcValue } from "./allowed-types";
 
-export interface MessageChannel<Message extends JsonValue | void = void> {
+export interface MessageChannel<Message extends IpcValue | void = void> {
```

Comment:
Nit: New types introduced here add behaviour, making this PR something else than refactoring :)

Reply:
Well it is more of back tracking the behaviour that was removed when these types were introduced.